packages/package_picard_1_135/.shed.yml

```diff
@@ -0,0 +1,3 @@
+owner: devteam
+name: package_picard_1_135_0
+    
```

Comment:
Feel free to refuse this request too, but it'd be nice if this could be filled out like the IUC does. It'd help us automatically upload it. E.g. https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_bedtools_2_19/.shed.yml and then copy the metadata from the existing picard package in the TS.